tests/web/test_web_client_coverage.py

```diff
@@ -631,6 +631,49 @@ async def test_coverage(self):
                 elif method_name == "admin_conversations_restrictAccess_removeGroup":
                     self.api_methods_to_call.remove(method(channel_id="D123", group_id="G123", team_id="T13")["method"])
                     await async_method(channel_id="D123", group_id="G123", team_id="T123")
+                elif method_name == "admin_conversations_create":
```

Comment:
Your tests don't cover new ones. Please modify `all_api_methods` in the way the comment describes.